### PR TITLE
Event Listeners Page

### PR DESCRIPTION
**Pull request by carlos-logro**
issue: #668 

# Changes

First pass for the last component in the trigger trio.
Also added a correction in `TriggerBinding` reducer.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: src/containers/EventListeners/EventListeners.js [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+import { fetchEventListeners } from '../../actions/eventListeners';
+import {
+  getEventListeners,
+  getEventListenersErrorMessage,
+  getSelectedNamespace,
+  isFetchingEventListeners,
+  isWebSocketConnected
+} from '../../reducers';
+
+import '../TriggerTemplates/TriggerTemplates.scss';
```
Comment (a-roberts) [inline src/containers/EventListeners/EventListeners.js]: Hmm what's the deal with this? Shouldn't we have a shared scss across all related components or at least EventListeners.scss?

**Comment by carlos-logro** [comment]
Created a scss folder with the intention of keeping generic styles and added the trigger one per @a-roberts request .

File: src/containers/EventListeners/EventListeners.js [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+        })
+      }
+    ];
+
+    const eventListenersFormatted = eventListeners.map(listener => ({
+      id: `${listener.metadata.namespace}:${listener.metadata.name}`,
+      name: listener.metadata.name,
+      namespace: listener.metadata.namespace,
+      date: (
+        <FormattedDate date={listener.metadata.creationTimestamp} relative />
```
Comment (AlanGreene) [inline src/containers/EventListeners/EventListeners.js]: The header for this column is 'Date Modified', I suspect we may have this error on some of the other recently added pages too

File: src/reducers/index.js [hunk 3]
```diff
@@ -30,6 +31,7 @@ import taskRuns, * as taskRunsSelectors from './taskRuns';
 
 export default combineReducers({
   clusterTasks,
+  eventListeners: eventListeners(),
```
Comment (AlanGreene) [inline src/reducers/index.js]: should just be `eventListeners,` for consistency with all of the other reducers

File: src/containers/EventListeners/EventListeners.js (hunk 2 above)
Comment (a-roberts) [reply src/containers/EventListeners/EventListeners.js]: Making a followup issue for this, yeah it's on the others too

File: src/containers/EventListeners/EventListeners.js [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+          defaultMessage: 'Namespace'
+        })
+      }
+    ];
+
+    const eventListenersFormatted = eventListeners.map(listener => ({
+      id: `${listener.metadata.namespace}:${listener.metadata.name}`,
+      name: listener.metadata.name,
+      namespace: listener.metadata.namespace,
+      date: (
```
Comment (AlanGreene) [inline src/containers/EventListeners/EventListeners.js]: The header for this column is 'Date Modified', I suspect we may have this error on some of the other recently added pages too

**Pull request merged**